frontend/src/app/Settings/Facility/FacilityForm.tsx

```diff
@@ -37,9 +39,10 @@ export const useFacilityValidation = (facility: Facility) => {
         clearError(field);
         await facilitySchema.validateAt(field, facility);
       } catch (e) {
-        setErrors((existingErrors) => ({
-          ...existingErrors,
-          [field]: allFacilityErrors[field],
+        let error = createFieldError(field, facility);
+        setErrors((errors) => ({
```

Comment:
Changing this naming to `errors` causes shadowing. The previous code uses the naming `existingErrors` as `errors` is defined as a variable within the scope of `useFacilityValidation`

Reply:
Good point, changed back to `existingErrors`. (Also renamed my new `error` to `errorMessage` as I think that's a bit clearer)